Gems/ROS2/Code/CMakeLists.txt

```diff
@@ -67,6 +67,9 @@ ly_add_target(
             Gem::StartingPointInput
             Gem::PhysX.Static
             Gem::LmbrCentral.API
+            Gem::ImGui.imguilib
+        RUNTIME_DEPENDENCIES
+                Gem::ImGui.imguilib
```

Comment:
Is it needed? Already imgui is added to ROS2 gem.

Reply:
Moved the file